plugins/action/zos_unarchive.py

```diff
@@ -118,6 +118,13 @@ def run(self, tmp=None, task_vars=None):
                         task_vars=task_vars,
                     )
                 )
+                if format_name in USS_SUPPORTED_FORMATS:
```

Comment:
Move this to also when unarchive action fails.

Reply:
I looked this over and the change was made, since Fernando is on holiday tomorrow, I think its fair to resolve this one on his behalf.